Exchange-Guide.md

```diff
@@ -31,6 +31,15 @@ Default is `matching` mode, which means that `xud` acts as order book & matching
 * `SubscribeSwaps` informs about successful/failed swaps with other peers in an event stream and is to be used with `PlaceOrder`. Use this call to get real-time swap events from placed orders to track order executions, update your user's balances, and inform users when one of their orders was settled.
 * `ExecuteSwap` executes a swap for an match your internal matching engine found with a remote peer. This call requires `orderID`, `pairID` & `quantity`. You can continue using your internal `orderID`s, `xud` takes care of mapping such with `orderID`s in the Exchange Union network. This is a sync call, returning the final result of a swap. It might take several seconds for the swap to complete.
 
+### Notes
+* `xud` doesn't *forward* orders, it only sends to and receives from direct peers. Reason for this, among others, are manipulation and speed. A more scalable approach will be [introduced in a later stage](https://github.com/ExchangeUnion/Docs/blob/master/XU-TechnicalPaper.md#xus-dob-protocol-has-the-following-goals).
+* Restarting or stopping `xud` clears all orders. When `xud` boots up, it automatically reconnects to its peers and fetches all open orders.
+* `xud` relies on is it's own (local) view of orders. This is also how it determines if it acts as a maker or a taker when a match with a peer order is found. If the peer order reached the order book first it acts as taker, if the local order reached the order book first it acts as maker. The swap protocol defines that only as taker, `xud` actively initiates a swap, as maker it'll wait for the swap to be initiated by the remote taker. For the same reason, `xud` never matches incoming peer orders with the existing order book; only incoming local orders. Due to network latency it now can happen that both parties determine to be maker in a match. In this case, a swap won't be initiated - nothing happens.
+* To query existing orders one can use `ListOrders`, in addition to the `existing` flag for the `SubscribeAddedOrders` streaming call.
+* Peer discovery and auto-connecting to new peers is still experimental and only enabled gradually. In order to manually connect to a peer, use `connect [NodeURI]` where `NodeURI` is `pubkey@IP:Port`, e.g. `02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6@xud1.test.exchangeunion.com:8885`.
+* gRPC is locally exposed on port 8886 by default. If you encounter a `grpc_status 14` error, check your SSL setup.
```

Comment:
I'd say "The gRPC API is locally exposed..." Also do you have more details about the 14 error?

Reply:
done, deleted the error 14 hint since I don't remember how exactly I encountered this